src/js/actions/index.js

```diff
@@ -542,3 +543,19 @@ export const setShouldRebuildFilterParams = (flag = true) => (dispatch) => {
     type: FILTER_FORM_PARAMS_BUILT,
   });
 };
+
+export const addLines = (workflow, lines, id) => dispatch => dispatch({
```

Comment:
```suggestion
export const addLines = ({ workflow, lines, id }) => dispatch => dispatch({
```
If you have many arguments it is in my opinion better to wrap it inside an object, so you don't need to follow the order while passing the arguments.
Treat it as an advice for the future or a titbit, not a change request if you don't wanna deal with refactoring it now.